engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -454,17 +459,40 @@ public void setErrorMessage(String errorMessage)
       this.errorMessage = errorMessage;
     }
 
+    public String getLogFileName()
+    {
+      return logFileName;
```

Comment:
There should be no setter for fileName and offset, only one setter for logFileInfo